exercicios/para-casa/entregas/amanda-santos/AuthorExample.js

```diff
@@ -0,0 +1,35 @@
+class AuthorExample {
+   
+    constructor(name, email, birthday){
+        this.name = name;
+        this.email = email;
+        this.birthday = birthday;
+
+    }
+
+    getName(){
+        return this.name;
+    }
+
+    getEmail(){
+        return this.email;
+    }
+
+    getBirthday(){
+        return this.birthday;
+    }
+
+    setName(name){
+        this.name = name
+    }
+
+    setEmail(email){
+        this.email = email;
+    }
+
+    setBirthday(birthday){
+        this.birthday = birthday;
+    }
+}
+
```

Comment:
Senti falta da implementação dos set / get atributos de acesso e modificadores que são escrito confome o ex:
`set name(newNome){
this.#nome = newNome
}

get nome() {
return this.#nome
}